CHANGELOG.rst

```diff
@@ -14,6 +14,11 @@ Change Log
 Unreleased
 ----------
 
+[0.65.3] - 2018-02-06
+---------------------
+
+* Refactor the code.
```

Comment:
We need a meaningful sentence here, otherwise we should just stop using this file all together.